docs/configuration/config-file-reference.md

```diff
@@ -1689,6 +1689,11 @@ cassandra:
   # CLI flag: -cassandra.query-concurrency
   [query_concurrency: <int> | default = 0]
 
+  # Table options used to create index or chunk tables.(Default = "": use
+  # default table options of Cassandra
+  # CLI flag: -cassandra.table-with
+  [table_with: <string> | default = ""]
```

Comment:
Be sure to update https://github.com/cortexproject/cortex/blob/master/docs/production/storage-cassandra.md with information about this flag. As well as warnings about the potential issues a user could face by misconfiguring it.

Reply:
@jtlisi @coelho
I have written the description about `table_options` to the `storage-cassandra.md`.
I think I am not good at English...
Please check if there is anything strange about the contents.

Reply:
LGTM